build/pxc-entrypoint.sh

```diff
@@ -471,6 +471,12 @@ if [ -z "$CLUSTER_JOIN" ] && [ "$1" = 'mysqld' ] && [ -z "$wantHelp" ]; then
 		fi
 		set -x
 
+		echo "CREATE FUNCTION IF NOT EXISTS get_last_record_timestamp_by_binlog RETURNS INTEGER SONAME 'binlog_utils_udf.so'" | "${mysql[@]}"
+
+		echo "CREATE FUNCTION IF NOT EXISTS get_gtid_set_by_binlog RETURNS STRING SONAME 'binlog_utils_udf.so'" | "${mysql[@]}"
+
+		echo "CREATE FUNCTION IF NOT EXISTS get_first_record_timestamp_by_binlog RETURNS INTEGER SONAME 'binlog_utils_udf.so'" | "${mysql[@]}"
```

Comment:
i see a lot of test failures with the error `full cluster crash detected`. probably these statements are the reason. also do we have these in mysql 5.7?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/2264/commits/e9d086fe3ac08f49f65b7a844ee26fa104c0da55